### ISSUE TITLE
Fix metric code

### ISSUE THREAD

**Issue by dseidman** [BrewTroller/BrewTroller-Official#16]
Right now the metric code for temperature just divides the degrees Fahrenheit by 2! This is seriously inaccurate so I have to believe that nobody is using the metric code. This should be an easy fix. I suspect there are similar issues with liters/gallons elsewhere in the code.

**Comment by jvetter1** [comment]
@dseidman Can you provide more info on this issue? We are looking to fix it for 2.8 but you have given us very little to go on. The code for calculating metric temperature is as follows:

From Temp.cpp:

```c++
#ifdef USEMETRIC
    return int(tempOut); 
#else
    return int((tempOut * 9 / 5) + 3200);
#endif
```
Note that conversion only happens for Fahrenheit not Celsius because the device provides temps in Celsius.

We would appreciate it if you could write back more details on where the conversion is bad. Thanks!

**Comment by dseidman** [comment]
This bug is 18 months old, so it's probably obsolete. It looks like the related code has been substantially refactored since I filed it. I'll go ahead and close it.
